src/Models/Address.php

```diff
@@ -5,38 +5,29 @@
 namespace Tipoff\Addresses\Models;
 
 use Tipoff\Support\Models\BaseModel;
+use Tipoff\Support\Traits\HasCreator;
 use Tipoff\Support\Traits\HasPackageFactory;
+use Tipoff\Support\Traits\HasUpdater;
 
 class Address extends BaseModel
 {
+    use HasCreator;
     use HasPackageFactory;
```

Comment:
The Address model doesn't have a factory yet. Can you add it to this PR, @djamesfar?

Reply:
Thank you